VMware.CloudFoundation.PasswordManagement.psd1

```diff
@@ -10,7 +10,7 @@
     RootModule        = '.\VMware.CloudFoundation.PasswordManagement.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '1.8.2.1004'
+    ModuleVersion     = '1.8.2.1005'
```

Comment:
No change needed here since this has no changes in the `.psm1` file and is limited to the tests.

Reply:
Resolved in 9727035.